messages/scratchOrgSettingsGenerator.md

```diff
@@ -0,0 +1,4 @@
+# noCapitalizeRecordTypeConfigVar
+
+Record types defined in the scratch org definition file will stop being capitalized by default in a future release.
+Set the `org-capitalize-record-types` config var to `true` to enforce capitalization.
```

Comment:
this warning will be visible to both projects using sfdx-core and the CLI.

For lib users, I wanted to make this an object option rather than the config (that way the config var would live in plugin-org) but decided against it (more about that below).